src/main/scala/com.snowplowanalytics/weather/providers/darksky/Responses.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2015-2018 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.weather
+package providers.darksky
+
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.JsonCodec
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.generic.extras.semiauto.{deriveUnwrappedDecoder, deriveUnwrappedEncoder}
+
+object Responses {
+
+  implicit val unitsDecoder: Decoder[Units] = deriveUnwrappedDecoder[Units]
+  implicit val unitsEncoder: Encoder[Units] = deriveUnwrappedEncoder[Units]
+
+  implicit val flagsEncoder: Encoder[Flags] = deriveEncoder[Flags]
```

Comment:
do we need encoders?

Reply:
Absolutely, didn't know `@JsonCodec` macro takes parameters to exclude decoders/encoders.